base/base.bzl

```diff
@@ -48,79 +49,107 @@ def distro_components(distro_suffix):
                 workdir = workdir,
             )
 
-    container_image(
-        name = "base" + distro_suffix,
-        base = ":static_root_amd64" + distro_suffix,
-        debs = [
-            DISTRO_PACKAGES["amd64"][distro_suffix]["libc6"],
-            DISTRO_PACKAGES["amd64"][distro_suffix]["libssl1.1"],
-            DISTRO_PACKAGES["amd64"][distro_suffix]["openssl"],
-        ],
-    )
+            container_image(
+                name = "base_" + user + "_" + arch + distro_suffix,
+                architecture = arch,
+                base = ":static_" + user + "_" + arch + distro_suffix,
+                debs = [
+                    DISTRO_PACKAGES[arch][distro_suffix]["libc6"],
+                    DISTRO_PACKAGES[arch][distro_suffix]["libssl1.1"],
+                    DISTRO_PACKAGES[arch][distro_suffix]["openssl"],
+                ],
+            )
 
-    # A debug image with busybox available.
-    container_image(
-        name = "debug" + distro_suffix,
-        base = ":base" + distro_suffix,
-        directory = "/",
-        entrypoint = ["/busybox/sh"],
-        env = {"PATH": "$$PATH:/busybox"},
-        tars = ["//experimental/busybox:busybox.tar"],
-    )
+            # A debug image with busybox available.
+            container_image(
+                name = "debug_" + user + "_" + arch + distro_suffix,
+                architecture = arch,
+                base = ":base_" + user + "_" + arch + distro_suffix,
+                directory = "/",
```

Comment:
Maybe `directory` is unnecessary, as it's [default](https://github.com/bazelbuild/rules_docker#container_image-1) is `/`?